site/tutorials/keptn-multistage-qualitygates-08.md

```diff
@@ -26,6 +26,10 @@ The full setup that we are going to deploy is sketched in the following image.
 
 ![demo setup](./assets/keptn-multistage-podtatohead/demo-workflow.jpg)
 
+If you are interested, please have a look at this article that explains the deployment in more detail.
+
+![https://dev.to/gabrieltanner/modern-continuous-delivery-on-kubernetes-for-developers-5chf](.)
```

Comment:
Please update to a valid link

[Modern continuous delivery on Kubernetes for Developers - dev.to](https://dev.to/gabrieltanner/modern-continuous-delivery-on-kubernetes-for-developers-5chf)